### PR TITLE
Some work in Customers

### PR DESCRIPTION
**Pull request by ShiroMx**

### CODE REVIEW

File: Projects/TEAM-2/GroupProject/GroupProject/Customer.cs [hunk 1]
```diff
@@ -7,7 +7,31 @@
 
 namespace GroupProject
 {
-    public class Customer
+    public class Customers
```
Comment (LoveMyDaughter) [inline Projects/TEAM-2/GroupProject/GroupProject/Customer.cs:10]: Customer

File: Projects/TEAM-2/GroupProject/GroupProject/Customer.cs [hunk 2]
```diff
@@ -7,7 +7,31 @@
...
     {
+       public string Name;
+       public float money;
+
+       public Customers(string Name, float money)
```
Comment (LoveMyDaughter) [inline Projects/TEAM-2/GroupProject/GroupProject/Customer.cs:15]: Customer

File: Projects/TEAM-2/GroupProject/GroupProject/Customer.cs [hunk 3]
```diff
@@ -7,7 +7,31 @@
...
+        }
+
+    public float Set_money(float money)
+        {
+            this.money = money;
+
+            return money.ToString();
+        }
+
+     Customers new_customer = new Customers();
```
Comment (LoveMyDaughter) [inline Projects/TEAM-2/GroupProject/GroupProject/Customer.cs:35]: It's better to create customers in the Program class. Delete or comment this line.

**Review by LoveMyDaughter — CHANGES_REQUESTED** [review]

File: Projects/TEAM-2/GroupProject/GroupProject/Customer.cs (hunk 1 above)
Comment (ShiroMx) [reply Projects/TEAM-2/GroupProject/GroupProject/Customer.cs]: Edit

File: Projects/TEAM-2/GroupProject/GroupProject/Customer.cs (hunk 2 above)
Comment (ShiroMx) [reply Projects/TEAM-2/GroupProject/GroupProject/Customer.cs]: Complete

File: Projects/TEAM-2/GroupProject/GroupProject/Customer.cs (hunk 3 above)
Comment (ShiroMx) [reply Projects/TEAM-2/GroupProject/GroupProject/Customer.cs]: Complete

**Review by LoveMyDaughter — APPROVED** [review]
ok

**Pull request merged**